We don't have indexmap 2.x in debian yet.

The version was bumped upstream in
https://github.com/QEDK/configparser-rs/commit/2a6e75c35e9f960970cf4bcba14db622ff10b5e1

Index: configparser/Cargo.toml
===================================================================
--- configparser.orig/Cargo.toml
+++ configparser/Cargo.toml
@@ -38,7 +38,7 @@ version = "1.12.0"
 optional = true
 
 [dependencies.indexmap]
-version = "2.1.0"
+version = ">= 1.9.1"
 optional = true
 
 [badges.maintenance]
